src/test/java/com/solvd/laba/carina/ProductsTest.java

```diff
@@ -0,0 +1,24 @@
+package com.solvd.laba.carina;
+
+import com.solvd.laba.carina.products.gui.pages.*;
+import com.solvd.laba.carina.products.utils.TestUtil;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ProductsTest implements IAbstractTest {
+
+    @Test
+    @MethodOwner(owner = "hhan")
+    public void productsTest() {
+        HomePage homePage = new TestUtil().openHomePage();
+
+        ProductsPage productsPage = homePage.clickProductsButton();
+        Assert.assertTrue(productsPage.isPageOpened(), "Products page is not opened");
```

Comment:
AbstractPage also has method assertPageOpened that performs the same assertion with set pageOpeningStrategy

Reply:
Yes, I have switched all the manual asserts to assertPageOpened.